src/lite-yt-embed.js

```diff
@@ -17,7 +17,7 @@ class LiteYTEmbed extends HTMLElement {
         // Gotta encode the untrusted value
         // https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-2---attribute-escape-before-inserting-untrusted-data-into-html-common-attributes
         this.videoId = encodeURIComponent(this.getAttribute('videoid'));
-
+        this.startTime = encodeURIComponent(this.getAttribute('startTime'));
```

Comment:
Wouldn't we want to `parseInt` the value? That would still give a clear direct value being filtered for security while also validating what we give to the embed URL is a number.